be/src/common/config.h

```diff
@@ -294,6 +294,8 @@ CONF_Int32(update_compaction_per_tablet_min_interval_seconds, "120"); // 2min
 CONF_mInt64(max_update_compaction_num_singleton_deltas, "1000");
 CONF_mInt64(update_compaction_size_threshold, "268435456");
 CONF_mInt64(update_compaction_result_bytes, "1073741824");
+// Timeout of waiting for apply after compaction commit, counters in ms.
+CONF_mInt64(update_compaction_wait_for_version_timeout_ms, "120000"); // 120s
```

Comment:
If you and a timeout are configured, how to control the memory usage during 120s?

Reply:
We can increase `update_compaction_wait_for_version_timeout_ms`, let compaction wait apply finish, so compaction's apply task won't be pending in apply queue.